frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices-mode/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -202,6 +203,7 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
           {!!nodesCount && (
             <SelectNodesDetails cpu={cpu} memory={memory} zones={zones.size} nodes={nodesCount} />
           )}
+          <EnableTaintNodes state={state} dispatch={dispatch} mode="ATTACHED_DEVICES" />
```

Comment:
We already have `mode` passed down from main component.
Check `install.tsx`. We can pass that down and use it here.